hbase-server/src/main/resources/META-INF/services/org.apache.hadoop.hbase.security.provider.SaslServerAuthenticationProvider

```diff
@@ -0,0 +1,3 @@
+org.apache.hadoop.hbase.security.provider.DigestSaslServerAuthenticationProvider
+org.apache.hadoop.hbase.security.provider.GssSaslServerAuthenticationProvider
+org.apache.hadoop.hbase.security.provider.SimpleSaslServerAuthenticationProvider
```

Comment:
Unused imports?

Reply:
These automatically register our server-side parameters so we never have to fiddle with the configuration (and users can't unregister our the implementations we expect to be there: SIMPLE, KERBEROS, and delegation tokens/DIGEST-MD5).